### PR TITLE
[IMP]10719 add module invoice_report.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: invoice_report/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{
+    'name': 'Invoice Report',
+    'version': '1.0',
+    'category': '',
+    'depends': ['stock', 'sale'],
+    'author': 'Elico Corp',
+    'license': 'AGPL-3',
+    'website': 'https://www.elico-corp.com',
+    'description': """
+        Module which hides the OpenERP announcement bar.
```
Comment (elicoidal) [inline invoice_report/__openerp__.py]: ? description is folkloric...

File: invoice_report/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import invoice_report
```
Comment (elicoidal) [inline invoice_report/__init__.py]: Header/README/PEP8

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

File: invoice_report/README.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
+'description': """
```
Comment (elicoidal) [inline invoice_report/README.rst]: @Rona111 What is this?
You need to check the modules first, not only copy paste.
README is completely wrong.

**Comment by elicoidal** [comment]
What is the purpose of the module?
Create a new invoice report?
@duanyp1991 @siyuancn @seb-elico should we move forward or cancel?

File: invoice_report/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Invoice Report',
+    'version': '1.0',
```
Comment (siyuancn) [inline invoice_report/__openerp__.py]: version is wrong

File: invoice_report/company_view.xml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+	<data>
+		<record id="company_form_detail_extend" model="ir.ui.view">
```
Comment (siyuancn) [inline invoice_report/company_view.xml]: view id is: view_company_form

File: invoice_report/invoice_report.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class accountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    discount_amount = fields.Float(
```
Comment (siyuancn) [inline invoice_report/invoice_report.py]: Compute Field: the compute method pattern is _compute_<field_name>

File: invoice_report/invoice_report.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+        string='Discount Amount', compute='_discount_amount')
+    original_amount = fields.Float(
+        string='Original Amount', compute='_original_amount')
```
Comment (siyuancn) [inline invoice_report/invoice_report.py]: Compute Field: the compute method pattern is _compute_<field_name>

**Comment by siyuancn** [comment]
@Rona111 Please make sure the purpose of the module first.

File: invoice_report/README.rst (hunk 3 above)
Comment (elicoidal) [reply invoice_report/README.rst]: Update to newest template

**Review by elicoidal — CHANGES_REQUESTED** [review]
Update template/short headers/openerp.py

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Review by elicoidal — APPROVED** [review]

**Pull request merged**